apps/profiles/viewsets.py

```diff
@@ -0,0 +1,42 @@
+
+from rest_framework import viewsets
+
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+
+from apps.profiles.serializers import (
+    UserRatingListSerializer,
+    UserRatingUpdateSerializer,
+    UserRatingCreateSerializer,
+    UserRatingRetrieveSerializer,
+)
+from apps.profiles.permissions import UserRatingPermissions
+from apps.profiles.models import UserRating
+
+
+class UserRatingViewSet (viewsets.ModelViewSet):
+    """
+    A simple ViewSet for viewing and editing user's ratings.
+    """
+    model = UserRating
+    queryset = UserRating.objects.all()
+    lookup_url_kwarg = 'rating_id'
+    permission_classes = [IsAuthenticatedOrReadOnly, UserRatingPermissions]
+    http_method_names = ["post", "get", "put", "delete"]
+
+    def get_queryset(self):
```

Comment:
As far as I remember you either define queryset attribute or get_queryset method. you should choose one of the two.